test/module/irohad/multi_sig_transactions/gossip_propagation_strategy_test.cpp

```diff
@@ -79,8 +79,11 @@ PropagationData subscribeAndEmit(boost::optional<PropagationData> data,
   EXPECT_CALL(*pbfactory, createPeerQuery())
       .WillRepeatedly(testing::Return(boost::make_optional(
           std::shared_ptr<iroha::ametsuchi::PeerQuery>(query))));
+  iroha::GossipPropagationStrategyParams gossip_params;
+  gossip_params.period = period;
```

Comment:
Since fields have different type it's ok to use just initializer list, eg:
`iroha::GossipPropagationStrategyParams gossip_params{period, amount};`

Same for other places

Reply:
I do not like it in such form, when you do not specify the names of members explicitly. If somebody changes the structure and forgets to check all such initializations, he might introduce an error.

Reply:
But the C90 variant of explicit member initialization is wonderful, and I wait for it in C++20 (as you pointed out above).

Reply:
> If somebody changes the structure and forgets to check all such initializations, he might introduce an error.

The types are different, so that probably won't be happened

Reply:
And what about using adding same types, or having implicit conversions? I would rather avoid that risks.